src/v1/__tests__/test.ts

```diff
@@ -577,6 +619,32 @@ test.each(cityResolverResponsesV20221101)(
   }
 );
 
+test.each(cityResolverResponsesV20230901)(
```

Comment:
Consider adding negative test cases to ensure the robustness of the new changes. This can help in catching potential bugs and issues. [important]